frontend/src/app/game/game/game.component.ts

```diff
@@ -26,62 +27,59 @@ export class GameComponent implements AfterViewInit {
               return `${formatCZK(+value / 1_000_000, false)} mil.`;
             }
 
-            return`${formatCZK(+value / 1_000_000_000, false)} mild.`;
+            return `${formatCZK(+value / 1_000_000_000, false)} mild.`;
           },
         },
       }],
     },
   };
 
-  private _infectedToday$ = new Subject<ChartValue>();
-  infectedToday$ = this._infectedToday$.asObservable();
+  infectedToday$: Observable<ChartValue> = this.gameService.gameState$.pipe(map(gameState => ({
+    label: gameState.date,
+    value: gameState.stats.detectedInfections.today,
+    tooltipLabel: (value: number) => `Nově nakažení: ${formatCZK(value, false)}`,
+  })));
 
-  private _costTotal$ = new Subject<ChartValue>();
-  costTotal$ = this._costTotal$.asObservable();
+  costTotal$: Observable<ChartValue> = this.gameService.gameState$.pipe(map(gameState => ({
+    label: gameState.date,
+    value: gameState.stats.costTotal,
+    tooltipLabel: (value: number) => `Celkové náklady: ${formatCZK(value)}`,
+  })));
 
-  private _deathToday$ = new Subject<ChartValue>();
-  deathToday$ = this._deathToday$.asObservable();
+  deathToday$: Observable<ChartValue> = this.gameService.gameState$.pipe(map(gameState => ({
+    label: gameState.date,
+    value: gameState.stats.deaths.today,
+    tooltipLabel: (value: number) => `Nově zemřelí: ${formatCZK(value, false)}`,
+  })));
 
-  private _immunized$ = new Subject<{resistant: ChartValue; vaccinated: ChartValue}>();
-  immunized$ = this._immunized$.asObservable();
+  immunized$: Observable<ChartValue[]> = this.gameService.gameState$.pipe(map(gs => ([
+    {
+      label: gs.date,
+      value: Math.round(gs.sirState.resistant) + gs.stats.vaccinationRate * Math.round(gs.sirState.suspectible),
+      tooltipLabel: (value: number) => `Imunizováni kumulativně: ${formatCZK(value, false)}`,
+    },
+    {
+      label: gs.date,
+      value: gs.stats.vaccinationRate * Math.round(gs.sirState.suspectible),
+      tooltipLabel: (value: number) => `Vakcinovaní: ${formatCZK(value, false)}`,
+      datasetOptions: {
+        backgroundColor: `${colors.critical}33`,
+        borderColor: `${colors.warn}`,
+      },
+      color: colors.warn,
+    },
+  ])));
 
-  costTotalTooltip(value: number) {
-    return `Celkové náklady: ${formatCZK(value)}`;
-  }
+  resistant$ = this.gameService.gameState$.pipe(
```

Comment:
Not sure about the names `immunized$` vs. `resistant$`, here. Would not `immunized$` and `immunizedDetail$` and `immunizedChart$` or be better?